### ISSUE TITLE
Use Kvantum's scrollbars

### ISSUE THREAD

**Issue by CDrummond** [MartinBriza/adwaita-qt#94]
Both AdwaitaQt and [Kvantum](https://github.com/tsujan/Kvantum) have support for overlay scrollbars. In AdwaitaQt this seems to be achieved by just not drawing the scrollbar until the mouse is over it - i.e. the view still has the space reserved for the scrollbar (which is noticable with tree views that have a header - this causes the header to not be drawn fully to the edge)

Kvantum on the other-hand seems to now have actual overlay scrollbars - where no space is reserved, and they are drawn over the view when required. Perhaps the code from Kvantum could be use in AdwaitaQt?

**Comment by MartinBriza** [comment]
Yeah, I noticed some themes have that, too. I guess implementing it in a similar way would be a pretty good idea.
I'm going to look into it at some point for sure. Now I'm focusing on making Wayland support better and creating QtQuick themes. After that is done, I'm going to fix this and some bugs which may appear in the meantime (especially after the OmgUbuntu post, which raised the interest in this theme by a huge margin).

**Comment by heroin-moose** [comment]
Bump?

**Comment by MartinBriza** [comment]
@heroin-moose I'm really sorry but I'm not going to work on this in the near future. :/ Definitely leaving this open so in the future, I (or somebody else) can fix this.

**Comment by heroin-moose** [comment]
:(
